src/Components/Modals/DeleteActivationKeyConfirmationModal.js

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import {
+  Button,
+  Modal,
+  ModalVariant,
+  TextContent,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import propTypes from 'prop-types';
+import useDeleteActivationKey from '../../hooks/useDeleteActivationKey';
+import useNotifications from '../../hooks/useNotifications';
+import Loading from '../LoadingState/Loading';
+import { useQueryClient } from 'react-query';
+
+const DeleteActivationKeyConfirmationModal = (props) => {
+  const { isOpen, handleModalToggle, name } = props;
+  const { addSuccessNotification, addErrorNotification } = useNotifications();
+  const { mutate, isLoading } = useDeleteActivationKey();
+  const queryClient = useQueryClient();
+
+  const deleteActivationKey = (name) => {
+    mutate(name, {
+      onSuccess: (data, name) => {
```

Comment:
linter is telling me to prefix `data` with an underscore since it's not used.
`_data`